### ISSUE TITLE
Wish to sync LLNG to github

### ISSUE THREAD

**Issue by whlsxl** [LemonLDAPNG/lemonldap-ng#1]
Thank you to the team for developing LemonLDAP-NG.

LLNG is hosted on https://gitlab.ow2.org/lemonldap-ng/lemonldap-ng/. It's not easy to report bugs & contribute. 

That would be great if this repository could be synchronized on GitHub and handle issues from both sides.

**Comment by coudot** [comment]
It would be harder for us to manage issues, we prefer to keep the tracker on OW2.

**Comment by e-Collectivites** [comment]
Hi Clément, +1 for the sync in Github, at least just the code without issues if possible?

As stated in readme : "If you'd like to have a copy of repository here, open an issue here 😉" ... so i have hope : ))

Thanks !

Romain

**Comment by Dushistov** [comment]
Is https://gitlab.ow2.org/lemonldap-ng/lemonldap-ng/ still working? I see only timeouts when trying to open it.

**Comment by coudot** [comment]
Yes repository on ow2 gitlab is working and is still the official one
